hack/local-up-cluster.sh

```diff
@@ -71,6 +71,8 @@ CLOUD_PROVIDER=${CLOUD_PROVIDER:-""}
 CLOUD_CONFIG=${CLOUD_CONFIG:-""}
 FEATURE_GATES=${FEATURE_GATES:-"AllAlpha=false"}
 STORAGE_BACKEND=${STORAGE_BACKEND:-"etcd3"}
+# preserve persistent stuff such as etcd. you also need to set ETCD_DIR.
+PRESERVE="${PRESERVE:-false}"
```

Comment:
nit: maybe `PRESERVE_ETCD`? otherwise this is pretty ambiguous.

Reply:
updated, thanks for quick review